library/system/io.lean

```diff
@@ -3,17 +3,17 @@ Copyright (c) 2016 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Luke Nelson, Jared Roesch and Leonardo de Moura
 -/
-constant {u} io : Type u → Type u
-constant io.functor : functor io
-constant io.monad   : monad io
+meta constant {u} io : Type u → Type u
+meta constant io.functor : functor io
```

Comment:
It is not clear to me whether we want `io` to be `meta`. 
On one hand, it makes senses, we cannot prove anything about programs using `io`, and it is wrapping C++ code.
On the other hand, it will be awkward to write a function in `io` and tag it as `meta`. Suppose we are writing a program, and its interface is implemented using `io`. It feels awkward to say the interface is `meta` (-programming). Perhaps, we need a different keyword (e.g., `untrusted`). What do you think?